docs/quickstart/basics.rst

```diff
@@ -8,46 +8,64 @@ Let's take a look at a very basic API using Flask-Rebar:
 
 .. code-block:: python
 
+   # todos.py
+
    from flask import Flask
    from flask_rebar import Rebar
-   from flask_rebar import ResponseSchema
```

Comment:
FWIW, I've never used `ResponseSchema` because I've always been the one in full control of my data, and have automated tests that make sure the data I keep in my database stays consistent (validation-wise) with the data my API promises to return. In the interest of keeping this example as basic as possible, while still being as useful as possible, I thought it's better to save Rebar's special `RequestSchema/ResponseSchema` for a later section, in return for demonstrating the other stuff I added here.

Reply:
I tend to agree on that one but I think we should move toward #161